docs/introduction.md

```diff
@@ -0,0 +1,35 @@
+---
+author: "Lewi Lie Uberg"
```

Comment:
@eliotwrobson Do we need the author specified for every md page?

Reply:
I wasn't sure, but I just followed the convention. But I agree that it seems a little excessive, given that we have credits per-person in the mkdocs. I'll go ahead and remove.

Reply:
Removed here: https://github.com/caleb531/automata/pull/189/commits/ff3a43b2c559070a9ed0f68337d3f0ec16834e20